app/src/main/java/edu/wpi/first/shuffleboard/app/components/WidgetPane.java

```diff
@@ -193,14 +194,20 @@ public TileSize sizeOfWidget(Widget widget) {
             (int) (height / getTileSize()));
   }
 
-  public void removeWidget(WidgetTile tile) {
+  /**
+   * Remove a given tile from the widget pane.
```

Comment:
This should have a `@return` tag because what this returns is not obvious without looking at the source

Reply:
fixed.